docs/projects/automate-project-git-source.ipynb

```diff
@@ -0,0 +1,2069 @@
+{
```

Comment:
<p>Use the code files from your project context folder on your local machine (<code>local=True</code>) can be used for debug purposes.  - maybe worth to mention that we will want to use it for debugging the code on the user local machine</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/3640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
updated